buildSrc/build.gradle

```diff
@@ -116,7 +116,7 @@ dependencies {
   api 'com.avast.gradle:gradle-docker-compose-plugin:0.14.12'
   api 'org.apache.maven:maven-model:3.6.2'
   api 'com.networknt:json-schema-validator:1.0.36'
-  api 'com.fasterxml.jackson.core:jackson-databind:2.12.5'
+  api 'com.fasterxml.jackson.core:jackson-databind:2.12.6'
```

Comment:
Could you please change it to`${props.getProperty('jackson')}` ? Thank you.

Reply:
@reta Sure I will change that